test/unit/test_devices/test_aireos_device.py

```diff
@@ -0,0 +1,522 @@
+import pytest
+from unittest import mock
+
+from pyntc.devices import AIREOSDevice
+from pyntc.devices.aireos_device import (
+    CommandError,
+    OSInstallError,
+    CommandListError,
+    FileTransferError,
+    RebootTimeoutError,
+    NTCFileNotFoundError,
+    convert_filename_to_version,
+)
+
+
+@pytest.mark.parametrize(
+    "filename,version",
+    (("AIR-CT5520-K9-8-8-125-0.aes", "8.8.125.0"), ("AIR-CT5520-8-8-125-0.aes", "8.8.125.0")),
+    ids=("encrypted", "unencrypted"),
+)
+def test_convert_filename_to_version(filename, version):
```

Comment:
Why not a TestAireosDevice class? I think this would be a better pattern and consistent with our other tests.